src/coreclr/vm/gdbjit.cpp

```diff
@@ -1301,8 +1301,8 @@ void FunctionMember::DumpLinkageName(char* ptr, int& offset)
     namespaceOrClassName.ConvertToUTF8(utf8namespaceOrClassName);
     methodName.ConvertToUTF8(utf8methodName);
```

Comment:
This can be deleted and `GetUTF8` can be called in `namespaceOrClassName` and `methodName` directly.